templates/page.html

```diff
@@ -432,10 +432,11 @@ <h3>Near-term</h3>
         <div class="card">
           <div class="card-body">
             <h5 class="card-title">
-              <a href="https://github.com/orgs/reanahub/projects/50" class="card-link">Workspace file retention</a>
+              <a href="https://github.com/reanahub/reana-client-go" class="card-link">Go client and stable API</a>
             </h5>
             <p class="card-text">
-                Introduce user-configurable workspace file retention policy allowing to specify that some files are to be deleted automatically 1 day after workflow finishes, other files retained for 356 days, etc. <a href="https://github.com/reanahub/reana-client/issues/610">[rfc]</a>
+              Introduce new command-line client written in Go, to improve performance and ease of use. At the same time, stabilize the current REST API to guarantee compatibility between clients and REANA.
```

Comment:
At the same time... -> We could perhaps shorten to: "At the same time, stabilise the REST API versioning for future evolution."